### PR TITLE
docs: add bucket module examples to the function guide 

### PR DESCRIPTION
**Pull request by TolgaYigit**

### CODE REVIEW

File: docs/content/guide/function.md [hunk 1]
```diff
@@ -169,6 +170,205 @@ export default async function() {
...
+
+###### Get
+
+```typescript
+export default function (req, res) {
+  Bucket.initialize({apikey: "{APIKEY}"});
+  return Bucket.get("{BUCKET ID}").then(result => {
+    console.log(result);
+    res.send(result);
+  });
```
Comment (thesayyn) [inline docs/content/guide/function.md:197]:
```suggestion
  return Bucket.get("{BUCKET ID}");
```

File: docs/content/guide/function.md [hunk 2]
```diff
@@ -169,6 +170,205 @@ export default async function() {
...
+
+###### Bucket Data Get with Parameters
+
+```typescript
+export default function (req, res) {
+  Bucket.initialize({apikey: "{APIKEY}"});
+  return Bucket.data
+    .getAll("{BUCKET ID}", {
+      headers: {"accept-language": "TR"},
+      querParams: {paginate: true, skip: 1}
```
Comment (thesayyn) [inline docs/content/guide/function.md:310]:
```suggestion
      queryParams: {paginate: true, skip: 1}
```

File: docs/content/guide/function.md [hunk 3]
```diff
@@ -54,9 +54,10 @@ Spica provides modules to your function in runtime. Modules work like a module i
 
 In order to use these modules in a **function**, they need to be added as **dependency** on **Function Edit page**.
 
-| Module                   | Description                                                                                                  |
-| ------------------------ | ------------------------------------------------------------------------------------------------------------ |
-| `@spica-devkit/database` | This module has a public API for making database operations like **update**, **delete**, **create**, **get** |
+| Module                   | Description                                                                                                                     |
+| ------------------------ | ------------------------------------------------------------------------------------------------------------------------------- |
+| `@spica-devkit/database` | This module has a public API for making database operations like **update**, **delete**, **create**, **get**                    |
+| `@spica-devkit/bucket`   | This module has a public API for making both Bucket and Bucket Data operations like **update**, **delete**, **insert**, **get** |
```
Comment (thesayyn) [inline docs/content/guide/function.md:60]: Maybe we could add `@spica-devkit/dashboard` as well?

Comment (TolgaYigit) [reply docs/content/guide/function.md:60]: Then we have to move the content in dashboard.md to this document. Is it ok for you?

Comment (thesayyn) [reply docs/content/guide/function.md:60]: I am not sure about that. I think we better leave it as is for now

**Pull request merged**